src/effects/deeplearning/ModelManagerPanel.cpp

```diff
@@ -196,7 +197,8 @@ void ModelManagerPanel::SetSelectedCard(ModelCardHolder card)
 // ManagerToolsPanel
 
 ManagerToolsPanel::ManagerToolsPanel(wxWindow *parent, ModelManagerPanel *panel)
-   : wxPanelWrapper((wxWindow *)parent, wxID_ANY, wxDefaultPosition, wxSize(managerPanel_w, 30))
+   : wxPanelWrapper((wxWindow *)parent, wxID_ANY, wxDefaultPosition, wxSize(static_cast<int>(getScreenWidth()/cardPanel_w) + static_cast<int>(getScreenWidth()/detailedCardPanel_h) + 
```

Comment:
two liner pls

Reply:
removing this from the PR, making separate PR for UI